src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/FtpaAppellantDecisionOutcomeType.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum FtpaAppellantDecisionOutcomeType {
+
+    FTPA_GRANTED("granted"),
+    FTPA_PATIALLY_GRANTED("partiallyGranted"),
```

Comment:
I think we have a typo here. Should be FTPA_PARTIALLY_GRANTED possibly?

Reply:
Done